docs/src/content/docs/components/badges.mdx

```diff
@@ -0,0 +1,156 @@
+---
+title: Badges
+description: Learn how to use badges in Starlight.
```

Comment:
The other description gives a small indication of what you'd use the component for, something like this maybe?

```suggestion
description: Learn how to use badges in Starlight to display additional pieces of information.
```